sdks/java/io/amazon-web-services/src/main/java/org/apache/beam/sdk/io/aws/options/AwsModule.java

```diff
@@ -112,7 +112,9 @@ public AWSCredentialsProvider deserializeWithType(
         JsonParser jsonParser, DeserializationContext context, TypeDeserializer typeDeserializer)
         throws IOException {
       Map<String, String> asMap =
-          jsonParser.readValueAs(new TypeReference<Map<String, String>>() {});
+          checkNotNull(
```

Comment:
In this case `checkNotNull` is fine 🤯 
I don't get it why strings are treated differently by checker

Reply:
I'm tempted to just reenable `@SuppressWarnings({"nullness"})` for the entire class 😳

Reply:
Seems yes, it might be a solution